.github/workflows/ci.yml

```diff
@@ -383,6 +383,10 @@ jobs:
           include:
             - { modules: core/trino-main }
             - { modules: testing/trino-tests }
+            - { modules: testing/trino-tests, profile: test-fault-tolerant-hive-1 }
+            - { modules: testing/trino-tests, profile: test-fault-tolerant-hive-2 }
+            - { modules: testing/trino-tests, profile: test-fault-tolerant-delta }
+            - { modules: testing/trino-tests, profile: test-fault-tolerant-iceberg }
```

Comment:
@nineinchnick @findepi 
I am concerned this will probably run for each build on CI. Should we have separate, more isolated module for that?
Or separate modules for hive/delta/iceberg?

Reply:
What's the condition when these tests should be run, if not always?

Reply:
if trino-main/spi/etc change all should run.
Otherwise if specific connector changes (change in hive connector should trigger test-*-hive*, ...)

Reply:
however, it would be nice not to run all the tests when eg only Iceberg changes...
i'm fine with several additional modules, if this pays off with CI execution times.

Reply:
Yeah I will extract. Single module for now `trino-recovery-tests`. Or should we go hard-core already and have separate for each connector?

Reply:
I will make one then - less pain - and should limit the burden already significantly.

Reply:
moved